src/suites/cts/command_buffer/render/render.spec.ts-disabled

```diff
@@ -0,0 +1,73 @@
+import { TestGroup } from '../../../../framework/index.js';
+import { GPUTest } from '../../gpu_test.js';
+
+export const g = new TestGroup(GPUTest);
```

Comment:
Should this be GPUShaderTest?

Reply:
Technically yes, but I don't really intend to keep the GPUShaderTest class separate in the long run, so whenever I turn these back on I'll deal with it.

Reply:
Actually, fixed, just in case anyone does try to use these between now and whenever I fix it.